include/picongpu/particles/atomicPhysics/debug/PrintAtomicDataToConsole.hpp

```diff
@@ -107,7 +107,8 @@ namespace picongpu::particles::atomicPhysics::debug
 
         // state data
         std::cout << "AtomicState Data" << std::endl;
-        std::cout << "index : [ConfigNumber, chargeState, levelVector]: E_overGround, IPDIonizationState[index, "
+        std::cout << "index : [ConfigNumber, chargeState, levelVector]: E_overGround, multiplicity, "
+                     "IPDIonizationState[index, "
                      "chargeState, configNumber]"
```

Comment:
Not blocking but there is an extra line break here.
```suggestion
                     "IPDIonizationState[index, chargeState, configNumber]"
```

Reply:
has been fixed in the IPD unit test PR